libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -397,6 +396,7 @@ bool NavEKF2_core::readDeltaVelocity(uint8_t ins_index, Vector3f &dVel, float &d
     if (ins_index < ins.get_accel_count()) {
         ins.get_delta_velocity(ins_index,dVel);
         dVel_dt = MAX(ins.get_delta_velocity_dt(ins_index),1.0e-4f);
+        dVel_dt = MIN(ins.get_delta_velocity_dt(ins_index),1.0e-1f);
```

Comment:
This should be `MIN(dVel_dt,1.0e-1f);`, right? Or with `dt`, if the one below is correct.